src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -99,8 +99,9 @@ private:                                                                    \
 // false, if we don't really care if the parameter is required or not.
 
 ////////////////////////////////////////////////////////////////////////////////
-#define COPY_TEXT_ARGS(X)                           \
-    X(bool, SingleLine, "singleLine", false, false) \
+#define COPY_TEXT_ARGS(X)                                       \
+    X(bool, DismissSelection, "dismissSelection", false, false) \

```

Comment:
The current behavior is "true". Should we keep the default as true?

Reply:
I set DismissSelection to false by default due to personal preference, but I agree with you, it should be set to true by default.

Reply:
Good start